packages/react-native-reanimated/src/layoutReanimation/web/componentUtils.ts

```diff
@@ -148,10 +148,12 @@ export function setElementAnimation(
 ) {
   const { animationName, duration, delay, easing } = animationConfig;
 
-  element.style.animationName = animationName;
-  element.style.animationDuration = `${duration}s`;
-  element.style.animationDelay = `${delay}s`;
-  element.style.animationTimingFunction = easing;
+  requestAnimationFrame(() => {
```

Comment:
If I understand correctly, we are delaying every animation start by one frame. Would it be better to only delay entering animations without layout changes and exiting animations?

Reply:
Good catch! Done in [b61d796](https://github.com/software-mansion/react-native-reanimated/pull/6172/commits/b61d79600bd2c8169a779e1807d127d0db0ea774)